### PR TITLE
Removed warnings plugin, updated Jenkins

### PR DESCRIPTION
**Pull request by alxsap**

### CODE REVIEW

**Comment by alxsap** [comment]
Waiting for the build

File: jenkinsfile-runner-base-image/packager-config.yml [hunk 1]
```diff
@@ -479,16 +479,11 @@ plugins:
...
-    artifactId: "warnings"
-    source:
-      version: "5.0.1"
-# Warnings Next Generation (Jenkins-Version: 2.204.4)
+# Warnings Next Generation (Jenkins-Version: 2.204.6)
   - groupId: "io.jenkins.plugins"
     artifactId: "warnings-ng"
     source:
-      version: "8.3.0"
+      version: "8.4.1"
```
Comment (srinikitha09) [inline jenkinsfile-runner-base-image/packager-config.yml:486]: I think we had problem with this version before right?

Comment (alxsap) [reply jenkinsfile-runner-base-image/packager-config.yml:486]: It was 8.4.0. My hope is that they fixed the issue in 8.4.1. Shall we give it a try?

**Comment by alxsap** [comment]
Pipeline tests successful

File: jenkinsfile-runner-base-image/packager-config.yml (hunk 1 above)
Comment (valiparsa) [reply jenkinsfile-runner-base-image/packager-config.yml]: FYI This issue is not yet resolved in 8.4.4

**Pull request merged**